### PR TITLE
chore(release): prepare release notes for OSS 22.04

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [x] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 1]
```diff
@@ -16,6 +16,33 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [Packaging] Improved default configuration for Debian packages
+
+#### Bug fixes
+
+- [API] Fixed PHP error messages on authentication APIs.
+- [API] Improved error codes and messages for the renewing password endpoint.
+- [Authentication] Added the possibility to define redirect uri for OpenID Connect authentication.
+- [Authentication] Fixed an LDAP authentication issue that made LDAP users in OU with special characters unable to authenticate.
+- [Authentication] Made error messages generic to improve security.
+- [Authentication] Removed extra spaces in endpoint definition.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:34]:
```suggestion
- [Authentication] Fixed a broken URL issue by removing extra spaces in endpoint definition.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 2]
```diff
@@ -17,6 +17,33 @@ If you have feature requests or want to report a bug, please go to our
...
+
+- [API] Fixed PHP error messages on authentication APIs.
+- [API] Improved error codes and messages for the renewing password endpoint.
+- [Authentication] Added the possibility to define redirect uri for OpenID Connect authentication.
+- [Authentication] Fixed an LDAP authentication issue that made LDAP users in OU with special characters unable to authenticate.
+- [Authentication] Made error messages generic to improve security.
+- [Authentication] Removed extra spaces in endpoint definition.
+- [Backup] Fixed errors with distant DBMS in logs.
+- [Configuration] Fixed a bug in trap relations with services by host groups.
+- [Configuration] Fixed deletion of images when importing a configuration on a Remote Server with Map installed.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:38]:
```suggestion
- [Configuration] Fixed the deletion of images when importing a configuration on a Remote Server with MAP installed.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 3]
```diff
@@ -16,6 +16,33 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [Backup] Fixed errors with distant DBMS in logs.
+- [Configuration] Fixed a bug in trap relations with services by host groups.
+- [Configuration] Fixed deletion of images when importing a configuration on a Remote Server with Map installed.
+- [Core] Fixed an error when accessing to the install wizard on new installation.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:38]:
```suggestion
- [Core] Fixed an issue when accessing the installation wizard for a new installation.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 4]
```diff
@@ -17,6 +17,33 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Core] Fixed an error when accessing to the install wizard on new installation.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:39]:
```suggestion
- [Core] Fixed an issue when accessing the installation wizard for a new installation.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 5]
```diff
@@ -16,6 +16,33 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [Install] Fixed an SQL error when installing a new platform using MySQL 8 as DBMS.
+- [Install] Removed a displayed error during update.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:40]:
```suggestion
- [Install] Removed a displayed error during an update.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 6]
```diff
@@ -17,6 +17,33 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Install] Fixed an SQL error when installing a new platform using MySQL 8 as DBMS.
+- [Install] Removed a displayed error during update.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:41]:
```suggestion
- [Install] Removed a displayed error during an update.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 7]
```diff
@@ -16,6 +16,33 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [LDAP] Fixed LDAP groups listing in contactgroups form.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:41]:
```suggestion
- [LDAP] Fixed LDAP groups listing in contact groups form.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 8]
```diff
@@ -17,6 +17,33 @@ If you have feature requests or want to report a bug, please go to our
...
+- [LDAP] Fixed LDAP groups listing in contactgroups form.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:42]:
```suggestion
- [LDAP] Fixed LDAP groups listing in contact groups form.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 9]
```diff
@@ -16,6 +16,33 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [ResourceStatus] Fixed an error when opening detail panel of a resource in downtime.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:42]:
```suggestion
- [ResourceStatus] Fixed an error when opening the detail panel of a resource in downtime.
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 10]
```diff
@@ -17,6 +17,33 @@ If you have feature requests or want to report a bug, please go to our
...
+- [ResourceStatus] Fixed an error when opening detail panel of a resource in downtime.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:43]:
```suggestion
- [ResourceStatus] Fixed an error when opening the detail panel of a resource in downtime.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 11]
```diff
@@ -16,6 +16,33 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [Server] Fixed a bug that prevented non-admin users to select host groups in some areas of Centreon (eg. Host Discovery mappers).
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:43]:
```suggestion
- [Server] Fixed a bug that prevented non-admin users from selecting Host Groups in some areas of Centreon (e.g. Host Discovery mappers).
```

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 12]
```diff
@@ -17,6 +17,33 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Server] Fixed a bug that prevented non-admin users to select host groups in some areas of Centreon (eg. Host Discovery mappers).
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:44]:
```suggestion
- [Server] Fixed a bug that prevented non-admin users from selecting Host Groups in some areas of Centreon (e.g. Host Discovery mappers).
```

File: i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md [hunk 13]
```diff
@@ -16,6 +16,33 @@ notre [Github](https://github.com/centreon/centreon/issues/new/choose).
...
+- [UX] Removed UI slowdown when browser has no internet access with CEIP enabled.
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md:44]: In 22.10, this one is in the Enhancements section.

File: versioned_docs/version-22.04/releases/centreon-core.md [hunk 14]
```diff
@@ -17,6 +17,33 @@ If you have feature requests or want to report a bug, please go to our
...
+- [UX] Removed UI slowdown when browser has no internet access with CEIP enabled.
```
Comment (smau-centreon) [inline versioned_docs/version-22.04/releases/centreon-core.md:45]: In 22.10, this one is in the Enhancements section.

**Review by jdelpierre — DISMISSED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by jdelpierre — DISMISSED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**